client/src/app/home/home/account-card/services/accounts.service.ts

```diff
@@ -22,6 +23,14 @@ export class AccountService {
     );
   }
 
+  public postAccount(data: AccountPost): Observable<any> {
+    return this.http.post('http://localhost:3000/accounts', data);
```

Comment:
add constant BASE_URL = 'http://localhost:3000/accounts' at the top of the file and reuse it for requests